inbox/api/srv.py

```diff
@@ -152,16 +152,21 @@ def ns_all():
 def _get_account_data_for_generic_account(data):
     email_address = data["email_address"]
     sync_email = data.get("sync_email", True)
+    smtp_server_host = data["smtp_server_host"] or "localhost"
```

Comment:
Did you run tests for this change and potentially add/modify them to cover this use case? I think these should be `data.get()`'s like the line above.

Reply:
I updated the lines and run the tests. They passed with no error. Sorry for the mistake. I was thought you just sending blank the SMTP values with keys.